go.mod

```diff
@@ -32,7 +32,7 @@ require (
 	github.com/cosmos/cosmos-proto v1.0.0-alpha8
 	github.com/cosmos/cosmos-sdk v0.46.16
 	github.com/cosmos/gogoproto v1.4.11
-	github.com/cosmos/ibc-go/v6 v6.3.0
+	github.com/cosmos/ibc-go/v6 v6.2.1
```

Comment:
For posterity, the previous celestia-app releases were on v6.2.0 but it looks like the change in gas costs was caused by 6.3.0 so 6.2.1 is fine.

Ref: https://github.com/cosmos/ibc-go/compare/v6.2.0...v6.2.1
Ref: https://github.com/celestiaorg/celestia-app/issues/3167#issuecomment-2090323504